profiles/tests/test_filter_profiles.py

```diff
@@ -129,14 +129,14 @@ def test_get_all_profiles_authorized_filter_activity_retail(self):
                        3 in response.data[i]["comp_activity"]]
         self.assertEqual(len(retail_type), filtered_len)
 
-    def test_get_all_profiles_authorized_filter_activity_HORACE(self):
-        response = self.client.get(f"/api/profiles/?activity_type=HORACE")
+    def test_get_all_profiles_authorized_filter_activity_HORECA(self):
+        response = self.client.get(f"/api/profiles/?activity_type=HORECA")
         filtered_len = len(response.data)
         self.assertEqual(200, response.status_code)
         response = self.client.get("/api/profiles/")
-        horace_type = [response.data[i]["comp_activity"] for i in range(len(response.data)) if
+        HORECA_type = [response.data[i]["comp_activity"] for i in range(len(response.data)) if
                        4 in response.data[i]["comp_activity"]]
-        self.assertEqual(len(horace_type), filtered_len)
+        self.assertEqual(len(HORECA_type), filtered_len)
```

Comment:
I _strongly_ recommend to do _explicit_ checks.

I guess you know what exactly number you expect to have here. Then use this number in the assert.

Why? Because this test will pass in case when it is actuall broken (let's say, you expected here to have number 2 both in expected and actual results, but got such errors, which leads to 0 in both cases).

It's really, really important — dont write asserts list `self.assertEqual(code_calculating_one_value(), code_calculating_another_value())`. Write asserts like `self.assertEqual(42, code_calculating_value())`, where 42 is the exact value you expect to have.

It's better to not have a unittest at all, than to have one, which cofuses you with wrong feeling like 'I have coverage here'.